src/odin/maneuversbuilder.cc

```diff
@@ -48,6 +48,8 @@ constexpr float kShortContinueThreshold = 0.6f;
 
 constexpr uint32_t kOverlayEdgeMax = 5; // Maximum number of edges to look for matching overlay
 
+constexpr float kUpcomingLanesThreshold = 3.f; // Kilometers
```

Comment:
I wonder if we will need to use duration based on speed with a minimum distance

Reply:
@danpaz are we updating this later?

Reply:
@dgearhart yeah, this is just a starting point. We can tweak this threshold later or even make it configurable in the future.